lib/ansible/modules/network/avi/avi_serviceenginegroup.py

```diff
@@ -495,95 +583,162 @@
+        type: int
     memory_per_se:
         description:
             - Amount of memory for each of the service engine virtual machines.
             - Default value when not specified in API or module is interpreted by Avi Controller as 2048.
+        type: int
     mgmt_network_ref:
         description:
             - Management network to use for avi service engines.
             - It is a reference to an object of type network.
+        type: str
     mgmt_subnet:
         description:
             - Management subnet to use for avi service engines.
+        type: dict
     min_cpu_usage:
         description:
             - When cpu usage on an se falls below the minimum threshold, virtual services hosted on the se may be consolidated onto other underutilized ses.
             - After consolidation, unused service engines may then be eligible for deletion.
             - Allowed values are 20-60.
             - Default value when not specified in API or module is interpreted by Avi Controller as 30.
+        type: int
     min_scaleout_per_vs:
         description:
             - Minimum number of active service engines for the virtual service.
             - Allowed values are 1-64.
             - Default value when not specified in API or module is interpreted by Avi Controller as 1.
+        type: int
     min_se:
         description:
             - Minimum number of services engines in this group (relevant for se autorebalance only).
             - Allowed values are 0-1000.
             - Field introduced in 17.2.13,18.1.3,18.2.1.
             - Default value when not specified in API or module is interpreted by Avi Controller as 1.
         version_added: "2.9"
+        type: int
     minimum_connection_memory:
         description:
             - Indicates the percent of memory reserved for connections.
             - Allowed values are 0-100.
             - Field introduced in 18.1.2.
             - Default value when not specified in API or module is interpreted by Avi Controller as 20.
         version_added: "2.9"
+        type: int
     minimum_required_config_memory:
         description:
             - Required available config memory to apply any configuration.
             - Allowed values are 0-90.
             - Field deprecated in 18.1.2.
             - Field introduced in 18.1.1.
         version_added: "2.9"
+        type: int
     n_log_streaming_threads:
         description:
             - Number of threads to use for log streaming.
             - Allowed values are 1-100.
             - Field introduced in 17.2.12, 18.1.2.
             - Default value when not specified in API or module is interpreted by Avi Controller as 1.
         version_added: "2.9"
+        type: int
     name:
         description:
             - Name of the object.
         required: true
+        type: str
+    nat_flow_tcp_closed_timeout:
+        description:
+            - Idle timeout in seconds for nat tcp flows in closed state.
```

Comment:
NAT, TCP, and UDP should all be capitalized in the description text (not, obviously, in the parameter names themselves).